config/jobs/kubernetes-sigs/cluster-api-provider-gcp/cluster-api-provider-gcp-ci.yaml

```diff
@@ -1,34 +1,42 @@
 periodics:
 - name: ci-cluster-api-provider-gcp-build
   interval: 1h
+  decorate: true
+  extra_refs:
+  - org: kubernetes-sigs
+    repo: cluster-api-provider-gcp
+    base_ref: master
+    path_alias: "sigs.k8s.io/cluster-api-provider-gcp"
   labels:
     preset-service-account: "true"
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
   spec:
     containers:
-    - image: gcr.io/k8s-testimages/gcloud-in-go:v20190125-cc5d6ecff3
-      args:
-      - "--repo=sigs.k8s.io/cluster-api-provider-gcp"
-      - "--root=/go/src"
-      - "--upload=gs://kubernetes-jenkins/logs"
-      - "--scenario=execute"
-      - "--"
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20191218-bcc4f6d-experimental
```

Comment:
Why experimental?

Reply:
That's what's used in the other jobs, I just cmd-c cmd-v :)